<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 205, 205);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>DATA BUKU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>371</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-radius: 5px;</string>
    </property>
    <property name="title">
     <string>Form Buku</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>ISBN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Judul</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editISBN">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="editJudul">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Penulis</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbGenre">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPenulis">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddGenre">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ICON/add.png</normaloff>../ICON/add.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddPenulis">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ICON/add.png</normaloff>../ICON/add.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Penerbit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Tgl Terbit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPenerbit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTerbit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>271</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string>Jumlah Stok</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editStok">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimpan">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ICON/add_book.png</normaloff>../ICON/add_book.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ICON/edit_book.png</normaloff>../ICON/edit_book.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ICON/delete_book.png</normaloff>../ICON/delete_book.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Judul Buku / Penulis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSearch">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid black;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ICON/search_book.png</normaloff>../ICON/search_book.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblBuku">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>711</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ISBN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Judul</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penulis</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penerbit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tgl Terbit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jumlah Stok</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
